+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Fri Jan  6 11:33:19 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL glusterfs
+ pkgs=glusterfs
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs glusterfs --node 1 --tempfile ''
+ tmpfile2=/tmp/tmph2q6eyab
+ '[' -z '' ']'
+ tmpfile=/tmp/tmph2q6eyab
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Fri Jan  6 11:33:52 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Fri Jan  6 11:33:53 2023 - INFO  - Start testing...
+ test_execution glusterfsd.service
+ service=glusterfsd.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-01-06 11:33:53'
+ test_restart glusterfsd.service
+ service=glusterfsd.service
+ systemctl restart glusterfsd.service
Failed to restart glusterfsd.service: Unit glusterfsd.service not found.
+ CHECK_RESULT 5 0 0 'glusterfsd.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service restart failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'glusterfsd.service restart failed'
+ LOG_ERROR 'glusterfsd.service restart failed'
+ message='glusterfsd.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service restart failed'
Fri Jan  6 11:33:54 2023 - ERROR - glusterfsd.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Fri Jan  6 11:33:54 2023 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status glusterfsd.service
+ grep 'Active: active'
Unit glusterfsd.service could not be found.
+ CHECK_RESULT 1 0 0 'glusterfsd.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'glusterfsd.service restart failed'
+ LOG_ERROR 'glusterfsd.service restart failed'
+ message='glusterfsd.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service restart failed'
Fri Jan  6 11:34:01 2023 - ERROR - glusterfsd.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Fri Jan  6 11:34:01 2023 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop glusterfsd.service
Failed to stop glusterfsd.service: Unit glusterfsd.service not loaded.
+ CHECK_RESULT 5 0 0 'glusterfsd.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service stop failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'glusterfsd.service stop failed'
+ LOG_ERROR 'glusterfsd.service stop failed'
+ message='glusterfsd.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service stop failed'
Fri Jan  6 11:34:02 2023 - ERROR - glusterfsd.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Fri Jan  6 11:34:03 2023 - ERROR - ../common/common_lib.sh line 39
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status glusterfsd.service
+ grep 'Active: inactive'
Unit glusterfsd.service could not be found.
+ CHECK_RESULT 1 0 0 'glusterfsd.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'glusterfsd.service stop failed'
+ LOG_ERROR 'glusterfsd.service stop failed'
+ message='glusterfsd.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service stop failed'
Fri Jan  6 11:34:09 2023 - ERROR - glusterfsd.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Fri Jan  6 11:34:10 2023 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start glusterfsd.service
Failed to start glusterfsd.service: Unit glusterfsd.service not found.
+ CHECK_RESULT 5 0 0 'glusterfsd.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service start failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'glusterfsd.service start failed'
+ LOG_ERROR 'glusterfsd.service start failed'
+ message='glusterfsd.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service start failed'
Fri Jan  6 11:34:11 2023 - ERROR - glusterfsd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Fri Jan  6 11:34:11 2023 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status glusterfsd.service
+ grep 'Active: active'
Unit glusterfsd.service could not be found.
+ CHECK_RESULT 1 0 0 'glusterfsd.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'glusterfsd.service start failed'
+ LOG_ERROR 'glusterfsd.service start failed'
+ message='glusterfsd.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service start failed'
Fri Jan  6 11:34:18 2023 - ERROR - glusterfsd.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Fri Jan  6 11:34:19 2023 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled glusterfsd.service
+ service=glusterfsd.service
++ systemctl is-enabled glusterfsd.service
Failed to get unit file state for glusterfsd.service: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Fri Jan  6 11:34:19 2023 - INFO  - Unit is indirect, ignoring.
+ grep -i 'fail\|error'
+ journalctl --since '2023-01-06 11:33:53' -u glusterfsd.service
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of glusterfsd.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of glusterfsd.service'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ systemctl start glusterfsd.service
Failed to start glusterfsd.service: Unit glusterfsd.service not found.
+ systemctl reload glusterfsd.service
Failed to reload glusterfsd.service: Unit glusterfsd.service not found.
+ CHECK_RESULT 5 0 0 'glusterfsd.service reload failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service reload failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'glusterfsd.service reload failed'
+ LOG_ERROR 'glusterfsd.service reload failed'
+ message='glusterfsd.service reload failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service reload failed'
Fri Jan  6 11:34:20 2023 - ERROR - glusterfsd.service reload failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_glusterfsd.sh line 33'
+ message='oe_test_service_glusterfsd.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_glusterfsd.sh line 33'
Fri Jan  6 11:34:21 2023 - ERROR - oe_test_service_glusterfsd.sh line 33
+ return 0
+ systemctl status glusterfsd.service
+ grep 'Active: active'
Unit glusterfsd.service could not be found.
+ CHECK_RESULT 1 0 0 'glusterfsd.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='glusterfsd.service reload causes the service status to change'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'glusterfsd.service reload causes the service status to change'
+ LOG_ERROR 'glusterfsd.service reload causes the service status to change'
+ message='glusterfsd.service reload causes the service status to change'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'glusterfsd.service reload causes the service status to change'
Fri Jan  6 11:34:22 2023 - ERROR - glusterfsd.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_glusterfsd.sh line 35'
+ message='oe_test_service_glusterfsd.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_glusterfsd.sh line 35'
Fri Jan  6 11:34:22 2023 - ERROR - oe_test_service_glusterfsd.sh line 35
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Fri Jan  6 11:34:23 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 8
+ test 8 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Jan  6 11:34:24 2023 - ERROR - failed to execute the case.
+ exit 8
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Jan  6 11:34:24 2023 - INFO  - start environment cleanup.
+ systemctl stop glusterfsd.service
Failed to stop glusterfsd.service: Unit glusterfsd.service not loaded.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmph2q6eyab ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmph2q6eyab
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Jan  6 11:34:38 2023 - INFO  - Finish environment cleanup!
